worker/deploy/enhanced-worker.js

```
@@ -0,0 +1,879 @@
+        case '/stats':
+          const stats = await worker.getStats();
+          return new Response(JSON.stringify(stats), {
+            headers: { 'Content-Type': 'application/json' }
+          });
+
+        case '/register':
+          if (request.method !== 'POST') {
+            return new Response('Method not allowed', { status: 405 });
+          }
+          
+          const registrationData = await request.json();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)